code/obj/machinery/computer.dm

```diff
@@ -77,6 +77,14 @@
 	///Special changes for deconstruction can be added by overriding this
 	proc/special_deconstruct(var/obj/computerframe/frame as obj)
 
+	proc/update_screen_image()
```

Comment:
Any chance this could be rolled up into update_icon()? (calling UpdateIcon() later on?)